lib/puppet/provider/firewalld_ipset/firewall_cmd.rb

```diff
@@ -24,8 +24,7 @@ def self.instances
       new(
         ensure: :present,
         name: ipset_id,
-        type: ipset_raw.match(/type: (.*)/)[1],
-        **options
+        type: ipset_raw.match(/type: (.*)/)[1], **options
```

Comment:
Not sure why the error went away, but the `**` operator is a ruby 2.0 construct.  I've opened a new PR where I've replaced this with a `merge`.